pkgs/tools/misc/yubikey-manager-qt/default.nix

```diff
@@ -85,6 +87,9 @@ mkDerivation rec {
   meta = with lib; {
     description = "Cross-platform application for configuring any YubiKey over all USB interfaces";
     homepage = "https://developers.yubico.com/yubikey-manager-qt/";
+    knownVulnerabilities = [
+      "The YubiKey Manager is no longer under active development, install `yubioath-flutter` instead."
```

Comment:
Is the plan to remove `yubikey-manager-qt` from nixpkgs after 25.05?  If so, the message should be explicit about it.

Reply:
```suggestion
      "Yubico has announced the End of Life of YubiKey Manager QT. Upstream recommends yubioath-flutter to configure a YubiKey. yubikey-manager-qt will be dropped after NixOS 25.05."
```